jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -1966,10 +2160,10 @@ Function Start-Migration {
 
             Write-ToLog -Message:('Updating UWP Apps for new user')
             $newUserProfileImagePath = Get-ItemPropertyValue -Path ('HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\ProfileList\' + $newusersid) -Name 'ProfileImagePath'
-            $path = $newUserProfileImagePath + '\AppData\Local\JumpCloudADMU'
-            If (!(test-path $path)) {
-                New-Item -ItemType Directory -Force -Path $path
-            }
+            # $path = $newUserProfileImagePath + '\AppData\Local\JumpCloudADMU'
```

Comment:
Why is this commented out?

Reply:
My fault, I removed it because the folder is being created in line 1892. I think it is good to have that extra validation